executor/load_data.go

```diff
@@ -306,39 +316,32 @@ func (e *LoadDataInfo) GetFieldsFromLine(line []byte) ([]string, error) {
 		sep = []byte(e.FieldsInfo.Terminated)
 	}
 	rawCols := bytes.Split(line, sep)
-	cols := escapeCols(rawCols)
-	return cols, nil
-}
-
-func escapeCols(strs [][]byte) []string {
-	ret := make([]string, len(strs))
-	for i, v := range strs {
-		output := escape(v)
-		ret[i] = string(output)
+	fields := make([]field, 0, len(rawCols))
+	for _, v := range rawCols {
+		f := field{v, false}
+		fields = append(fields, f.escape())
 	}
-	return ret
+	return fields, nil
 }
 
 // escape handles escape characters when running load data statement.
-// TODO: escape need to be improved, it should support ESCAPED BY to specify
```

Comment:
You can update here, but don't remove the TODO, ESCAPED BY is still not supported

Reply:
done